engines/graaljs/predict-url.js

```diff
@@ -15,14 +15,20 @@
 
 const predictFileName = (os) => {
 	switch (os) {
+		case 'mac64arm': {
+		  return 'macos-aarch64'
+		}
 		case 'mac64': {
-			return 'macos';
+			return 'macos-amd64';
 		}
 		case 'linux64': {
-			return 'linux';
+			return 'linux-amd64';
+		}
+	        case 'linux64arm': {
+			return 'linux-aarch64';
```

Comment:
Please fix the indentation

Reply:
sorry sorry for that @mathiasbynens does it now look ok for you? i am not sure why we get this long time not saw such issues i guess all files in this repo should get converted from tab to space that would help.